### PR TITLE
Couldn't sleep so updating profile

### PR DESCRIPTION
**Pull request by loftwah**
## Changes proposed

Updating my profile information.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

N/A

## Note to reviewers

Updating my profile to enable the new stuff.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2596"></a>

### CODE REVIEW

File: data/loftwah.json [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-      "name": "Facebook",
-      "url": "https://facebook.com/loftwah",
-      "icon": "FaFacebook"
-    },
-    {
-      "name": "Songs I've produced ",
+      "name": "Songs I've produced",
       "url": "https://www.youtube.com/watch?v=txk5S_r7vkg&list=PLKBAUoCO_FtkHiwRzyGzfhauIhNMBFw66",
       "icon": "FaYoutube"
     },
```
Comment (Panquesito7) [inline data/loftwah.json:30]: Shouldn't line 44 (Spotify icon) be updated to `FaSpotify` as well?

Comment (loftwah) [reply data/loftwah.json:30]: I noticed that, but I could see a `FaStpofy` that could be used. I am happy to change this, but it was already `spotify` previously when the others were updated automatically. I'm not sure if Spotify is supported by it.

Comment (eddiejaoude) [reply data/loftwah.json:30]: Fa Spotify icons are available, you can search on the icons page https://linkfree.eddiehub.io/icons

Screenshot 2023-01-02 at 18 49 58

File: data/loftwah.json [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
+  "displayStatsPublic": true,
   "bio": "DevOps Engineer, Music Producer, and big fan of Open Source.",
   "avatar": "https://github.com/loftwah.png",
+  "tags": [
+    "AWS",
+    "Beats",
+    "DevOps",
+    "Music",
+    "Open Source",
+    "Ruby",
```
Comment (eddiejaoude) [inline data/loftwah.json:13]: The trailing comma is not needed

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good, 2 small changes mentioned inline

File: data/loftwah.json (hunk 1 above)
Comment (loftwah) [reply data/loftwah.json:30]: Nice I'll fix this up today.

File: data/loftwah.json (hunk 2 above)
Comment (loftwah) [reply data/loftwah.json:13]: Cheers, Eddie! I wish JSON would have made things work this way 😅 I'll fix it up today.

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**